awx/main/analytics/collectors.py

```diff
@@ -33,6 +35,12 @@ def something(since):
 def config(since):
     license_info = get_license(show_key=False)
     return {
+        'platform': {
+            'system': platform.system(),
+            'dist': platform.dist(),
+            'release': platform.release(),
+            'kubernetes': 'KUBERNETES_SERVICE_PORT' in os.environ,
```

Comment:
@shanemcd @matburt do you think this is a reasonable way to detect k8s?  These presence of these env vars are about the only reliable mechanism *I* know of.

I don't know of any way to determine "kubernetes_type (OCP, EKS, AKS, GKE, )" _inside_ the container.

Reply:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/214912/55559476-48fea880-56bc-11e9-981d-d9adea3dea56.png">

@benthomasson you have any feedback on this?

Reply:
<img width="564" alt="image" src="https://user-images.githubusercontent.com/214912/55559830-09848c00-56bd-11e9-9e44-328bd9be28d4.png">

Reply:
`platform.type` could be `["traditional", "openshift", "k8s"]`

Reply:
This is useful information to capture. Thanks!